app/services/idv/inherited_proofing/mock/va_api_mock_client.rb

```diff
@@ -0,0 +1,50 @@
+module Idv::InheritedProofing
+  module Mock
+    class VaApiMockClient
+      VALID_AUTH_CODE = 'mocked-auth-code-for-testing'.freeze
+
+      USER_ATTRIBUTES = "{
+        first_name: 'Fakey',
+        last_name: 'Fakerson',
+        address: {
+          street: '123 Fake St',
+          street2: 'Apt 235',
+          city: 'Faketown',
+          state: 'WA',
+          country: nil,
+          zip: '98037'
+        },
+        phone: '2063119187',
+        birth_date: '2022-1-31',
+        ssn: '123456789'
+      }".freeze
+
+      def initialize(auth_code)
+        @auth_code ||= auth_code
+      end
+
+      def user_attributes
+        raise TypeError.new('Something went wrong!') if (@auth_code != VALID_AUTH_CODE)
```

Comment:
I think call to `new` is unnecessary; perhaps even a more descriptive error? 

`raise TypeError, "For shame! auth_code is invalid: #{@auth_code}" if @auth_code != VALID_AUTH_CODE`